tools/SdkVersionAnalyzer/DockerfileAnalyzer.cs

```diff
@@ -56,8 +55,8 @@ private static bool VerifySdkVersions(string content, DotnetSdkVersion expectedD
         // && echo "SHA256: $(sha256sum dotnet-install.sh)" \
         //     && echo "de4957e41252191427a8ba0866f640b9f19c98fad62305919de41bd332e9c820  dotnet-install.sh" | sha256sum -c \
         //     && chmod +x ./dotnet-install.sh \
-        //     && ./dotnet-install.sh -v 6.0.427 --install-dir /usr/share/dotnet --no-path \
-        //     && ./dotnet-install.sh -v 7.0.410 --install-dir /usr/share/dotnet --no-path \
+        //     && ./dotnet-install.sh -v 8.0.403 --install-dir /usr/share/dotnet --no-path \
```

Comment:
```suggestion
        //     && ./dotnet-install.sh -v 8.0.404 --install-dir /usr/share/dotnet --no-path \
```

Reply:
It is example of parsing. Not needed to upgrade version.